content/docs/how-to-guides/hasura-heroku-migration.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Migrate from Hasura with Heroku to Hasura with Neon 
+enableTableOfContents: true
+isDraft: true
+---
+
+This guide describes how to migrate your data from a Hasura with Heroku PostgreSQL to Hasuru with Neon PostgreSQL.
+
+It is assumed that you have already connected from Hasura Cloud to a Neon project. If not, refer [Running a Hasura App](/docs/integrations/hasura) for instructions.
+
+The instructions also assume that you have installed the Heroku CLI, which is used to transfer data from Heroku PostgreSQL to Neon PostgreSQL. For Heroku CLI installation instructions, see [The Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli).
+
+To migrate your Heroku data to Neon:
+
+1. [Retrieve your Heroku app name and database name](#retrieve-your-heroku-app-name-and-database-name)
+2. [Retrieve your Neon connection string](#retrieve-your-neon-connection-string)
+3. [Migrate your data](#migrate-your-data)
+4. [Verify that your data was transferred to Neon](#verify-that-your-data-was-transferred-to-neon)
+
+## Retrieve your Heroku app name and database name
+
+1. Log in to [Heroku](https://dashboard.heroku.com/) and select the project you want to migrate data from.
+1. Select **Overview** and copy the name of the Heroku Postgres database, which appears under **Installed add-ons**.
+2. Click **Settings** and copy your Heroku **App Name**.
+
+_**Note**_: You can also retrieve the Heroku Postgres database name by executing the following Heroku CLI command:
+
+```shell
+heroku pg:links --app <app>
+```
+
+where `<app>` is the Heroku App Name.
+
+For example:
+
+```shell
+$ heroku pg:links --app thawing-wave-57217
+=== postgresql-trapezoidal-48643
+```
+
+## Retrieve your Neon connection string
+
+You can find the Neon connection string in the Hasura Console or the Neon Console.
+
+To retrieve your Neon connection string from the Hasura Console:
+
+1. Log into your Hasura account.
+2. Select the project that you want to migrate and click **Launch Console**.
+3. Select the **DATA** tab.
+4. Copy the Neon connection string.
+
+To retrieve your Neon connection string from the Neon Console:
+
+1. Log in to the [Neon Console](https://console.neon.tech/app/projects).
```

Comment:
That method would not work, cause we do not expose the password

Reply:
Instead of select a project, we should create a project a copy the connection string

Reply:
Changed to creating a project.